cds/config.py

```diff
@@ -955,14 +955,25 @@ def _parse_env_bool(var_name, default=None):
 # OAuth
 ###############################################################################
 
-OAUTHCLIENT_REMOTE_APPS = dict(
-    cern=cern.REMOTE_APP,
+OAUTHCLIENT_CERN_OPENID_USERINFO_URL = (
```

Comment:
This is like the default, right? Maybe not needed?

Reply:
yes but we don't upgrade invenio-oauthclient so cannot rely on the default... unless I didn't understand your point :)